v03_pipeline/lib/annotations/gcnv.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+import datetime
+from typing import Any
+
+import hail as hl
+import pytz
+
+from hail_scripts.computed_fields import variant_id as expression_helpers
+
+
+def xpos(ht: hl.Table, **_: Any) -> hl.Expression:
+    # NB: `pos` is an aggregated field over samples.
```

Comment:
I think this comment is probably wrong in hindsight... the genotype entries annotations require a `hl.MatrixTable` so we should just go with `hl.MatrixTable`.   The conflict arrises when we join to other tables and need to know whether to reference `key` or `row_key`.